pkg/controller/installation/products/rhsso/reconciler.go

```diff
@@ -20,14 +22,16 @@ import (
 )
 
 var (
-	defaultRhssoNamespace   = "rhsso"
-	customerAdminPassword   = "Password1"
-	keycloakName            = "rhsso"
-	keycloakRealmName       = "openshift"
-	oauthId                 = "rhsso"
-	clientSecret            = "placeholder" // this should be replaced in INTLY-2784
-	defaultSubscriptionName = "integreatly-rhsso"
-	idpAlias                = "openshift-v4"
+	defaultRhssoNamespace               = "rhsso"
+	customerAdminPassword               = "Password1"
+	keycloakName                        = "rhsso"
+	keycloakRealmName                   = "openshift"
+	oauthId                             = "rhsso"
+	clientSecret                        = "placeholder" // this should be replaced in INTLY-2784
+	defaultSubscriptionName             = "integreatly-rhsso"
+	idpAlias                            = "openshift-v4"
+	githubIdpAlias                      = "github"
+	githubOauthAppCredentialsSecretName = "launcher-oauth-github"
```

Comment:
I am being really picky here (sorry).
The credentials aren't just for launcher(also for CodeReady), maybe don't call it "launcher-oauth-github"? :)

Reply:
Makes sense :smiley:  :+1: